### ISSUE TITLE
Content-Type not being set

### ISSUE THREAD

**Issue by stevestreza** [mattgemmell/MGTwitterEngine#34]
Reported by mathieut in mattgemmell/MGTwitterEngine#5. For almost all requests, the Content-Type header should be set to application/x-www-form-urlencoded. Currently, MGTwitterEngine doesn't set the Content-Type at all.

**Comment by mathieut** [comment]
I just tried with the last version of MGTwitterEngine you have here and I still get this error when I do my sendUpdate request with the accessToken I just got back:

2010-05-25 19:09:17.863 testTwitter[11239:207] Request failed for connectionIdentifier = 7642466B-9794-4626-96A0-E93371B31B12, error = Operation could not be completed. (HTTP error 401.) ((null))

I'm not using LibXML unlike taylanpince reported here: http://github.com/mattgemmell/MGTwitterEngine/issues#issue/5 but I'm using NSXML. I do have the last version of OAuthConsummer from jdg.
Everything seems to be fine in my Project: http://drp.ly/14B451

Maybe there's just no solution, and I'll wait until this famous OAuthCalypse and see what happens :(

Thanks anyway for your help.
